crates/sdk/src/install.rs

```diff
@@ -42,18 +41,18 @@ pub fn try_install_circuit_artifacts(artifacts_type: &str) -> PathBuf {
             build_dir.display()
         );
     } else {
-        cfg_if! {
-            if #[cfg(any(feature = "network", feature = "network-v2"))] {
-                println!(
-                    "[sp1] {} circuit artifacts for version {} do not exist at {}. downloading...",
-                    artifacts_type,
-                    SP1_CIRCUIT_VERSION,
-                    build_dir.display()
-                );
-                install_circuit_artifacts(build_dir.clone(), artifacts_type);
-            }
-        }
+        #[cfg(any(feature = "network", feature = "network-v2"))]
```

Comment:
note: no action needed, but I think it's kind of weird how the `network` feature is, in the rest of the sdk, responsible for the network prover, but now it's responsible for *accessing* the network for getting artifacts.

Reply:
Ohh youre saying that the artifacts actually shouldn't be gated because all modes mauy need them? 

Not actually sure just kept the original functionality here